### ISSUE TITLE
Animals 

### ISSUE THREAD

**Issue by Kurdistan2004** [SKHS-GDP2-S2B7-2019-2020/Prototype-2-Debugging-Challenge-S2B7#1]
Animals spawning outside of the board and that matters because the player cannot go through the boarder so the game thinks that you are avoiding the animals but your not the boarder is not letting you to go through so that needs to be fixed.

**Comment by DouglasUrner** [comment]
@Kurdistan2004 Does this seem to happen symmetrically (on both sides of the board) and with all of the animals?

Would it be better to expand the player's range of motion or to narrow the spawning area?

**Comment by Kurdistan2004** [comment]
> @Kurdistan2004 Does this seem to happen symmetrically (on both sides of the board) and with all of the animals?
> 
> Would it be better to expand the player's range of motion or to narrow the spawning area?

Yes It happens on both side of the board and yes all of the animals are like that it is just the animals outside of the boarder, I am gonna try to expand the player's range and see what happens.
